orchagent/vnetorch.cpp

```diff
@@ -1329,7 +1329,7 @@ VNetOrch::VNetOrch(DBConnector *db, const std::string& tableName, VNET_EXEC op)
 
     if (op == VNET_EXEC::VNET_EXEC_VRF)
     {
-        vr_cntxt = { VR_TYPE::ING_VR_VALID, VR_TYPE::EGR_VR_VALID };
+        vr_cntxt = { VR_TYPE::ING_VR_VALID, VR_TYPE::VR_INVALID };
```

Comment:
what is this VR_INVALID context? if we are using single vrf, why not just have one entry in the vr_cntxt?

Reply:
Removed VR_TYPE::VR_INVALID